app/main.js

```diff
@@ -5,6 +5,7 @@ var exported = {
   'child_process': null,
   'uuid': require('uuid'),
   'flowhub-registry': require('flowhub-registry'),
+  'fbp-protocol-client': require('fbp-protocol-client'),
```

Comment:
The connection is supposed to be owned by `runtime/ConnectRuntime` component, but right now we break encapsulation in the `noflo-runtime` element. That part should be refactored so that the Polymer element sends events to NoFlo-land and the correct component reacts to them